maistra/scripts/update-deps.sh

```diff
@@ -76,7 +76,7 @@ function copy_files() {
       fi
 
       cp -rL "${f}" "${VENDOR_DIR}" || echo "Copy of ${f} failed. Ignoring..."
-      echo "build --override_repository=${repo_name}=${VENDOR_DIR}/${repo_name}" >> "${BAZELRC}"
+      echo "build --override_repository=${repo_name}=/work/maistra/vendor/${repo_name}" >> "${BAZELRC}"
```

Comment:
should this be based on ${ROOT_DIR}?

Reply:
no. This is a hardcoded path `/work/...` that goes into `bazelrc`.